test/hotspot/jtreg/compiler/c2/irTests/TestMinMaxLongLoopBarrier.java

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
You might as well also find a better home for this test. We are trying to move away from `irTests`, and move tests to directories with the relevant "topic".